upgrade/init-3.4.4.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+if (!defined('_PS_VERSION_')) {
+    exit;
+}
+
+require_once _PS_MODULE_DIR_ . 'comfino/src/ConfigManager.php';
+
+/**
+ * @param Comfino $module
+ *
+ * @return bool
+ */
+function upgrade_module_3_4_4($module)
+{
+    $config_manager = new \Comfino\ConfigManager($module);
+
+    // Update code of widget initialization script.
+    $config_manager->updateWidgetCode('2928bbaf4ef9e8437a46c9f76eec4e90');
```

Comment:
Bezpieczny ten kod do udostępniania?

Reply:
To skrót MD5 poprzedniej domyślnej wersji skryptu startowego JS, aby sprawdzić, czy została zmieniona, aby nie nadpisać w aktualizacji nową domyślną wersją, starej z własnymi modyfikacjami użytkownika.